.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git*
+.dockerignore
+Dockerfile
```

Comment:
Are `Dockerfile` and `.dockerignore` not ignored by default?

Reply:
Not that I know of. But I moved it to the `v3.1` directory, since it is used as build context.